CHANGELOG.md

```diff
@@ -1,9 +1,17 @@
 # Changes to the Mapbox Navigation SDK for iOS
 
-## master
+## v0.18.1 (June 19, 2018)
+
+### Packaging
 
 * Increased the minimum deployment target of Core Navigation to iOS 9. ([#1494](https://github.com/mapbox/mapbox-navigation-ios/pull/1494))
+
+### User Interface
```

Comment:
Nit: use sentence case for headings. (“Core Navigation” is a proper noun, which is why it looks different.)